example/client/agent/main.go

```diff
@@ -140,6 +141,38 @@ func main() {
 		glg.Infof("%d - Results : %s\n\n", i+1, string(b))
 		time.Sleep(1 * time.Second)
 	}
+
+	glg.Info("Start removing vector")
+	// Remove indexed 400 vectors from vald cluster.
+	for i := range ids[:insertCount] {
+		// Call `Remove` function of Vald client.
+		// Sends id to server via gRPC.
+		_, err := client.Remove(ctx, &payload.Remove_Request{
+			// Conditions for removing the vector.
+			Id: &payload.Object_ID{
+				Id: ids[i],
+			},
+		})
+		if err != nil {
+			glg.Fatal(err)
+		}
+		if i%10 == 0 {
+			glg.Infof("Removed: %d", i+10)
+		}
+	}
+	glg.Info("Finish removing vector")
+	glg.Info("Start removing indexed vector from backup")
+	/**
+	Run Indexing instead of Auto Indexing.
+	Before running client.SaveIndex (or client.CreateAndSaveIndex), the vectors, which you called Remove API before, exist in NGT graph yet.
+	It is that the neighbor vectors will be returned when running SearchByID API even you called Remove API.
+	To delete removed vectors from NGT graph completely, client.SaveIndex will be used instead of waiting auto CreateIndex phase.
```

Comment:
```suggestion
	Before calling the SaveIndex (or CreateAndSaveIndex) API, the vectors you inserted before still exists in the NGT graph even the Remove API is called due to the design of NGT.
	So at this moment, the neighbor vectors will be returned from the SearchByID API.
	To completely remove the vectors from NGT graph completely, the SaveIndex API will be used here instead of waiting auto CreateIndex phase.
```

Reply:
```suggestion
        Before calling the SaveIndex (or CreateAndSaveIndex) API, the vectors you inserted before still exist in the NGT graph index even the Remove API is called due to the design of NGT.
	So at this moment, the neighbor vectors will be returned from the SearchByID API.
	To completely remove the vectors from the NGT graph completely, the SaveIndex API will be used here instead of waiting auto CreateIndex phase.
```

Reply:
Sorry it was my mistake. please remove `completely` here.
	To completely remove the vectors from the NGT graph completely, the SaveIndex API will be used here instead of waiting auto CreateIndex phase. 
->
	To remove the vectors from the NGT graph completely, the SaveIndex API will be used here instead of waiting auto CreateIndex phase.